src/components/ToolMenu/LayerTree/LayerTreeContextMenu/index.tsx

```diff
@@ -139,6 +159,28 @@ export const LayerTreeContextMenu: React.FC<LayerTreeContextMenuProps> = ({
     });
   };
 
+  const downloadLayer = (url: string) => {
+    if (!layer) {
+      return;
+    }
+    const reqOpts = {
+      method: 'GET',
+      headers: {
+        ...getBearerTokenHeader(client?.getKeycloak())
+      }
+    };
+    fetch(url, reqOpts)
+      .then((res) => {
+        return res.blob();
+      })
+      .then((blob) => {
+        const a = document.createElement('a');
+        a.href = URL.createObjectURL(blob);
+        a.setAttribute('download', layer.get('name'));
+        a.click();
+      });
```

Comment:
Minor, but can we switch to `async/await` instead?

Reply:
done, see 7e07cc76c5179b2f01be452d57a32fe7458148d9